protocols/migration/migration_reports_5_0_0_to_reports_6_0_0.py

```diff
@@ -41,6 +41,8 @@ def migrate_interpreted_genome_rd(self, old_instance):
         )
 
     def migrate_variants(self, old_variants):
+        if old_variants is None:
```

Comment:
Maybe return [] instead of None? Just means that code like the next line doesn't need to make an exception like this.

Reply:
You might need to explain that one for me again...